src/issues/currentIssue.ts

```diff
@@ -171,9 +183,12 @@ export class CurrentIssue extends Disposable {
 		return undefined;
 	}
 
-	private showBranchNameError(error: string) {
-		const editSetting = `Edit Setting`;
-		vscode.window.showErrorMessage(error, editSetting).then(result => {
+	private showBranchNameError(error: string, silent: boolean) {
```

Comment:
Same for this. I think if the branch name is bad then we should always show an error.